linux_os/guide/system/logging/log_rotation/ensure_logrotate_activated/ansible/shared.yml

```diff
@@ -0,0 +1,33 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = low
+
+- name: Configure daily log rotation in /etc/logrotate.conf
+  lineinfile:
+    create: yes
+    dest: "/etc/logrotate.conf"
+    regexp: "^daily$"
+    line: "daily"
+
+- name: Make sure daily log rotation setting is not overriden in /etc/logrotate.conf
+  lineinfile:
+    create: no
+    dest: "/etc/logrotate.conf"
+    regexp: "^(weekly|monthly|yearly)$"
```

Comment:
Unfortunately this is not working as expected. In case there is no "weekly" regexp starts passing, even if there is "monthly" or "yearly". I would rewrite it as loop. It is because we are asking for the regexp to be absent, not present.

Reply:
@vojtapolasek I could not reproduce the behavior you describe. I added a test scenario for it, could you check if it is as you describe?

Reply:
I am sorry, it turned out to be an issue with my screenreader which did not report indentations correctly to me. So it is fine. Thank you for the remediation.